integration/testnet/container.go

```diff
@@ -179,8 +179,8 @@ func (c *Container) bindPort(hostPort, containerPort string) {
 
 }
 
-// addFlag adds a command line flag to the container's startup command.
-func (c *Container) addFlag(flag, val string) {
+// AddFlag adds a command line flag to the container's startup command.
+func (c *Container) AddFlag(flag, val string) {
```

Comment:
Question: Why did we make this public?

Reply:
This snuck in from some testing I was doing updating now

Reply:
https://github.com/onflow/flow-go/pull/1763/commits/156ded703dcef22ed689085bad5a019152edb406